src/sentry/tasks/reports.py

```diff
@@ -704,6 +704,14 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
 
     user = User.objects.get(id=user_id)
 
+    if features.has("organizations:weekly-report-debugging", organization):
+        logger.info(
+            "reports.deliver_organization_user_report.begin",
+            extra={
+                "user": user.id,
```

Comment:
Hmmm, feels like this should be ID.

```suggestion
                "user_id": user.id,
```